src/components/options/settings.svelte

```diff
@@ -0,0 +1,113 @@
+<script>
+import _ from "lodash";
+import Notiflix from "notiflix";
+import { DateTime } from "luxon";
+import { onMount } from "svelte";
+
+import ConfigFile from "../../js/shared/config_file";
+import Fields from "../shared/fields.svelte";
+import ImportExport from "./settings/importExport.svelte";
+import LocalStore from "../../js/shared/local_store";
+import {
+  CONFIG_FILE_OPTIONS,
+  MERGE_OPTIONS,
+  SEARCH_RESULT_OPTIONS,
+  StoreKey,
+} from "../../js/shared/constants";
+
+// Bindings.
+let lastConfigUpdate;
+let lastConfigUpdateError;
+
+onMount(() => {
+  updateLastConfigUpdate();
+});
+
+function mainConfigurationUpdated(lazy) {
+  if (!lazy) {
+    updateLastConfigUpdate();
+  }
+
+  chrome.runtime.sendMessage({ action: "updateContextualMenu" });
+}
+
+async function updateLastConfigUpdate() {
+  var lastConfig = await LocalStore.getOne(StoreKey.LAST_CONFIG_DATA);
+
+  let date = _.get(lastConfig, "date", "-");
+  lastConfigUpdateError.innerHTML = _.get(lastConfig, "errorMsg", "");
+  lastConfigUpdate.innerHTML = DateTime.fromSeconds(date / 1000).toLocaleString(
+    DateTime.DATETIME_SHORT_WITH_SECONDS
+  );
```

Comment:
Now that this component has been converted to Svelte, you don't need to use `innerHtml`: instead you should use standard variabes, like:

```js
lastConfigUpdateError = _.get(lastConfig, "errorMsg", null);
lastConfigUpdate = DateTime.fromSeconds(date / 1000)...
```

````html
                <label class="mb-1">Last Updated on:</label>
                <div>{lastConfigUpdate}</div>
                {#if !!lastConfigUpdateError}
                     <div class="text-danger">{lastConfigUpdateError}</div>
                {/if}
```